src/data-workspace/data-entry-cell/use-active-cell.js

```diff
@@ -0,0 +1,19 @@
+import { useFormState } from 'react-final-form'
+import { useMetadata } from '../metadata-context.js'
+
+export const useActiveCell = () => {
+    const { active } = useFormState({ subscription: { active: true } })
+    const { metadata } = useMetadata()
+
+    const [deId, cocId] = active ? active.split('.') : [null, null]
+    const coIds = active
+        ? metadata.categoryOptionCombos[cocId].categoryOptions
```

Comment:
Is the COIds even needed? If so, please use a selector. `getCategoryOptionsByCoCId` should be what you need.